CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## v1.3.0 (04/03/2020)
+
+**Common**
+
+* Response headers are now stored in `olp::client::HttpResponse`. `olp::client::OlpClient::CallApi()` will collect and add the headers to the response.
+* X-Correlation-Id will be extracted from the HTTP response headers and used for subsequent requests in `olp::dataservice::read::StreamLayerClient`.
+* Added new backdown strategy to `olp::client::RetrySettings`.
+  The default implementation is `olp::client::ExponentialBackdownStrategy` which restricts the maximum wait
+  time during failed retries of network requests in `olp::network::client::OlpClient` class (both sync and async versions) in
+  a way that the accumulated wait time during retries shouldn't be longer than `RetrySettings.timeout` property.
+* Added `curl_global_init()` and `curl_global_cleanup()` to `olp::http::NetworkCurl` class (Linux default implementation).
+  Please refer to `olp::client::OlpClientSettingsFactory::CreateDefaultNetworkRequestHandler()` function for more information.
+* Fixed various warnings (MSVC, Clang, GCC). Users building with `-Wall -Wextra` should not face any hidden warnings now. This is now protected by a pre-release verification job.
+* Fixed a possible thread concurrency problem in `olp::client::OlpClient` class which in some situations resulted in a crash.
+* Reduced data copy in JSON serializers, which will increase performance.
+
+**olp-cpp-sdk-authentication**
+
+* A new API was added for generic federated authentication with a user-provided body.
+* Added a new API `olp::authentication::AuthenticationClient::IntrospectApp()` method which can be used to retrieve information
+  about the application associated with a particular access token.
+* A new `olp::authentication::AuthenticationClient` constructor was added using `olp::authentication::AuthenticationSettings` to setup network and
+  task scheduler. This is part of an ongoing authentication refactoring and users should switch to this constructor instead of using the
+  individual setters.
```

Comment:
```suggestion
  task scheduler. It is a part of an ongoing authentication refactoring. Switch to this constructor instead of using the individual setters.
```